plugins/hydra_ax_sweeper/README.md

```diff
@@ -0,0 +1,34 @@
+# Hydra Ax Launcher plugin
+
+This plugin provides a mechanism for Hydra applications to use the [Adaptive Experimentation Platform, aka Ax](https://ax.dev/). Ax can optimize any experiment - machine learning experiments, A/B tests, and simulations. 
+
+Run the following command to install the plugin:
+
+```
+pip install hydra_ax_sweeper
+```
+
+We include an example of how to use this plugin. The file [`plugins/hydra_ax_sweeper/example/banana.py`](plugins/hydra_ax/example/banana.py) implements the [Rosenbrock function (aka Banana function)](https://en.wikipedia.org/wiki/Rosenbrock_function). The return value of the function should be the value that we want to optimize.
+
+To compute the best parameters for the Banana function, clone the code and run the following command in the `plugins/hydra_ax_sweeper` directory:
+
+```
+python example/banana.py -m banana.x=-5:5 banana.y=-5:10.1
+```
+
+This sets the range of `x` parameter as `[-5, 5]` and the range of `y` parameter as `[-5, 10.1]`. Other supported formats are fixed parameters (eg `banana.x=5.0`) and choice parameters (eg `banana.x=1,2,3`). The values of the `x` and `y` parameters can also be set using the config file in `plugins/hydra_ax_sweeper/example/conf/config.yaml`. For instance, the configuration corresponding to the parameter `x` is as follows:
```

Comment:
mention that the configuration is optional and when you need it.

Reply:
The next line mentions that